http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```diff
@@ -156,16 +156,19 @@ final class LeoAppServiceInterp[F[_]: Parallel](
         if (diskResult.creationNeeded) F.pure(none[LastUsedApp])
         else {
           (diskResult.disk.formattedBy, diskResult.disk.galaxyRestore) match {
-            case (Some(FormattedBy.Galaxy), Some(galaxyRestore)) =>
+            case (Some(FormattedBy.Galaxy), Some(_)) | (Some(FormattedBy.Cromwell), None) =>
```

Comment:
What is the second argument in this case statement?

Reply:
`disk.galaxyRestore` stores details (PVC ids) on how to restore a Galaxy app specifically and reconnect to an existing disk that was formatted by Galaxy previously.

Reply:
FYI, apparently "restore" is used for the Pause/Resume functionality.